guide/getting-started/software.md

```diff
@@ -65,11 +67,12 @@ Primarily used by developers, command-line interfaces typically offer the most f
    layout = "float-right-desktop"
 %}
 
-If a blockchain can be thought of as a public database, then explorers are simply windows into that database. They typically display the data structure and maybe offer some basic functionality like looking up specific transactions to verify that they were processed.
+If blockchain is a public database, block explorers are simply windows into that database. They let users view data about specific transactions, latest blocks, block height, and so on.
```

Comment:
```suggestion
Block explorers are simply windows into the public Bitcoin blockchain. They let users view data about specific transactions, latest blocks, block height, and so on. Some block explorers such as [KYCP](https://kycp.org/#/) have more specific use cases such as tracking the privacy of your bitcoin UTXOs.
```

Expanded this paragraph slightly as currently on the website there is a big white space that would look a bit better filled.  https://bitcoin.design/guide/getting-started/software/#block-explorers

Reply:
Up to @GBKS if he wants this in.

Reply:
As this white space only exists on desktop, I would not add extra text just for that specific visual appearance.

There is another short paragraph below the image, we can just combine the two paragraphs to balance the columns.

Reply:
Done in https://github.com/BitcoinDesign/Guide/pull/437/commits/2a8065210dd4b367035f82cd0cbc35e444b39a95